sponsors.html

```diff
@@ -59,10 +59,15 @@ <h2>Silver Sponsors</h2>
       <section id="sponsor-bronze">
         <h2>Bronze Sponsors</h2>
         <p>
-          <a href="https://https://9elements.com" target="_blank" class="sponsor__logo">
+          <a href="https://9elements.com" target="_blank" class="sponsor__logo">
             <img src="./assets/sponsors/9elements_logo_black.svg" alt="9elments company logo">
           </a>
         </p>
+        <p>
+          <a href="https://flexvelop.com" target="_blank" class="sponsor__logo">
```

Comment:
The inline style is needed because the image has no width and height info.

In the `sponsor.html` I choose the `min-width` info so that images are connected near the top. See screenshot